addons/l10n_din5008_expense/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
nitpick: It is not needed I think

Reply:
It's indeed deprecated since 2012 in python 3 :p